src/hooks/useStore.ts

```diff
@@ -0,0 +1,56 @@
+import { create } from 'zustand'
+import { APIConfiguration } from '../../packages/common/types';
+
+
+export interface LandingConfigStore {
+  searchInput: string;
+  selectedTags: ReadonlyArray<string>;
+  view: string;
+  updateInput: (input: string) => void;
+  updateSelectedTags: (tagsArray: ReadonlyArray<string>) => void;
+  updateView: (newView: string) => void;
+}
+
+export const useLandingConfigStore = create<LandingConfigStore>()(
+  (set) => ({
+    searchInput: '',
+    selectedTags: [],
+    view: 'grid',
+    paginatedState: {count: 0, perPage: 10, page: 1},
```

Comment:
Is this repeated on the `pagination` store?

Reply:
Good catch!